app/views/DR/HomeScreen/DashboardCards.js

```diff
@@ -0,0 +1,52 @@
+import { Card, Text } from 'native-base';
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import Colors from '../../../constants/colors';
+import styles from './style';
+
+export default function dashboardCards({
```

Comment:
React Components must start with capital letter
```suggestion
export default function DashboardCards({
```